app/src/main/java/com/celzero/bravedns/ui/HomeScreenFragment.kt

```diff
@@ -276,16 +277,13 @@ class HomeScreenFragment : Fragment(R.layout.fragment_home_screen) {
         }
 
         b.fhsProxyChip.setOnCloseIconClickListener {
-            b.fhsProxyChip.isEnabled = false
-            appConfig.removeAllProxies()
-            b.fhsProxyChip.text = getString(R.string.hsf_proxy_chip_remove_text)
-            syncDnsStatus()
-            delay(TimeUnit.SECONDS.toMillis(2), lifecycleScope) {
-                b.fhsProxyChip.visibility = View.GONE
-                b.fhsProxyChip.isEnabled = true
-                showToastUiCentered(requireContext(),
-                                    getString(R.string.hsf_proxy_chip_removed_toast),
-                                    Toast.LENGTH_SHORT)
+            removeProxy()
+            io {
+                if(appConfig.isOrbotDns()) {
```

Comment:
`if<space>`

pls run the code formatter before check-in

Reply:
Done